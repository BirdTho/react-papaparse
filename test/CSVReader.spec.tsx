import React from 'react';
import expect from 'expect';
import CustomReader from './CustomReader';
import renderer from 'react-test-renderer';

// eslint-disable-next-line no-undef
describe('CSVReader', () => {
  // eslint-disable-next-line no-undef
  it('should match snapshot', function () {
    const handleFileUploaded = jest.fn().mockImplementation();
    const tree = renderer
      .create(
        <CustomReader
          onFileLoaded={handleFileUploaded}
          label="Chargez votre offre"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
