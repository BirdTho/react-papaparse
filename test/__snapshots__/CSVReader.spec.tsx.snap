// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSVReader should match snapshot 1`] = `
Array [
  <input
    accept="text/csv, .csv, application/vnd.ms-excel"
    onChange={[Function]}
    style={
      Object {
        "display": "none",
      }
    }
    type="file"
  />,
  <div
    onClick={[Function]}
    style={
      Object {
        "alignItems": "center",
        "borderColor": "#CCC",
        "borderRadius": 20,
        "borderStyle": "dashed",
        "borderWidth": 2,
        "cursor": "pointer",
        "display": "flex",
        "flexDirection": "column",
        "height": "100%",
        "justifyContent": "center",
        "padding": 20,
      }
    }
  >
    <span>
      Chargez votre offre
    </span>
  </div>,
]
`;
